src/node/genesis_gen.h

```diff
@@ -297,12 +315,12 @@ namespace ccf
     {
       auto service_view = tx.get_view(tables.service);
 
-      if (get_active_members_count() < get_recovery_threshold())
+      if (get_active_recovery_members().size() < get_recovery_threshold())
```

Comment:
Nit: `get_active_recovery_members()` is expensive, so would prefer:
```
const auto recovery_members_count = get_active_recovery_members().size();
if (recovery_members_count...)
{
  ..., recovery_members_count, ...
}
```

Reply:
Good spot.